src/Illuminate/Mail/Markdown.php

```diff
@@ -104,15 +103,13 @@ public function renderText($view, array $data = [])
      */
     public static function parse($text)
     {
-        $environment = Environment::createCommonMarkEnvironment();
-
-        $environment->addExtension(new TableExtension);
-
         $converter = new CommonMarkConverter([
             'allow_unsafe_links' => false,
-        ], $environment);
+        ]);
+
+        $converter->getEnvironment()->addExtension(new TableExtension());
```

Comment:
To help promote compatibility between the two major versions, `2.0.0-beta3` exposes an `Environment` here that can be used to add new extensions.  This is not the preferred way to add extensions, and it may be deprecated at some point, but it at least allows for easy compatibility by avoiding the `Environment` namespace changes.

Reply:
oooh, this was what I was missing. Thanks!